### DISCUSSION TITLE
Is it possibile to setup Redis as a store for aggressively cached documents?

### DISCUSSION THREAD

**Discussion by ynnova** [ravendb/ravendb#19118] [General]
The aggressively caching feature of RavenDb is one of my favourite feature of RavenDB.
From what I know the RAM of the client is used in order to store the cached documents.
It means that in a farm environment each node will have a copy of the cached document.

Is it possible to use a distributed query like Redis in order to have a centralized store for the cached documents?

**Comment by ayende** [comment]
No, it isn't possible. 
Adding Redis in this case isn't actually adding much to your performance, you'll still need to deal with network calls, but also now distributed cache state.

The whole point of the in memory cache is to reduce the network calls and bandwidth (queries in ravendb are usually really fast)

**Reply by ynnova** [reply]
Ok thank you.
